codejailservice/app.py

```diff
@@ -56,22 +54,41 @@ def configure_codejail(app):
                 limit_overrides_context=context,
             )
 
+
+# -----------------app configuration-------------------------
```

Comment:
This long kind of messages have a special syntax. See:  https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L57

Reply:
Yes, I changed taking care the django syntax `######### FEAT #############`. Also I have a codestyle issue E266 for to many `#`. 
![image](https://user-images.githubusercontent.com/51926076/141316974-c96a935e-9733-42a9-90f8-0825f76203b8.png)

Reply:
In this commit 2a4a46f ,  I can pass the pycodestyle given an extra whitespace. 
`# ################ FEAT ############` . What's your opinion about it?